### PR TITLE
feat: std versioning support

### LINKED ISSUE #5: provide `std` support
### Clear and concise description of the problem

It's often useful to also list the std version it uses.

### Suggested solution

provide `shield.deno.dev/std/` for compatible std version.

### Alternative

using [deno module](https://deno.land/x/std), however it displays as `deno.land/x` not `deno.land/std`.

### Additional context

I'm interested in opening a PR.

### PR DESCRIPTION
**Pull request by scarf005**
### Description

image

image

- resolves #5

### Additional context

- blocked by #8 (~~will be~~ was rebased after merge)
- refactored `index.html` and `mod.ts`

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -170,6 +186,9 @@ <h2 class="m-0 text-lg pt-4">Examples</h2>
 				<a href="https://shield.deno.dev/x/postgres" target="_blank">https://shield.deno.dev/x/postgres (module under 1.0)</a>
 				<img src="/x/postgres" alt="postgres deno module version"/>
 
+				<a href="https://shield.deno.dev/std/^1.35" target="_blank">https://shield.deno.dev/std/0.210.0 (std version)</a>
```
Comment (jeffydc) [inline index.html]: Thanks for the PR. Wrong URL here.
Also please use snake case for all variables and function names.

Comment (scarf005) [reply index.html]: alright. would the `snake_case` rule apply to `index.html`? or whole project-wide? i think `mod.ts` used `pascalCase`.

Comment (jeffydc) [reply index.html]: Snake case applies to the whole project.

Comment (scarf005) [reply index.html]: > Snake case applies to the whole project.

added it to `README.md` as it wasn't stated.

**Review by jeffydc — APPROVED** [review]
Thanks for the PR.

**Pull request merged**